guidance_plugin_validator/config/params.yaml

```diff
@@ -0,0 +1,37 @@
+# Copyright (C) 2021 LEIDOS.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+
+# Double: The time allocated for system startup. Validation begins after this much time has passed.
+# Units: s
+startup_duration: 60.0
+
+# Double: The time allocated for validation; once startup is complete, validation is complete after this much time has passed.
+# Units: s
+validation_duration: 60.0
+
+# List of String: Guidance Strategic Plugins that will be validated by this node
+guidance_strategic_plugins:
```

Comment:
Doe it make sense to make this an argument to the launch file more than a param? Or possibly both somehow (arg overriding param file?)? Just thinking, because that way we could set which plugins will be validated from the carma-config image - via the docker-compose.yml - which might be handy/convenient.

Reply:
Your suggestion definitely makes sense. I was unable to figure out a way to provide these arguments from the docker-compose.yml file, and instead added these as args within the carma_docker.launch file (in the related carma-config PR), so that a user can still add them within the carma-config image without re-building the image. Is this approach acceptable? 

Also, I have removed the parameters from the params.yaml file since I thought it may get confusing having two locations to input the lists of plugin names.